src/main/java/com/jalasoft/jfc/model/ImageConverter.java

```diff
@@ -0,0 +1,99 @@
+    //path where ImageMagick is installed.
+    protected String myPath = "C:\\Program Files (x86)\\ImageMagick-6.3.9-Q8\\";
+    //default value for creating thumbnail.
+    private final int THUMBNAILVALUE = 128;
+
+    /**
+     * convertImage Method por change a Image format, resize,
+     * rotate, blank and white, and create thumbnail.
+     * @param image variable of type Image.
+     * @return true if conversion is successful, false if fail.
+     */
+    protected boolean convertImage(ImageParam image) {
+        ProcessStarter.setGlobalSearchPath(myPath);
+        verifyDataValues(image);
+        boolean returnTrueOrFalse = false;
+        try {
+            ConvertCmd cmd = new ConvertCmd();
+            IMOperation op = new IMOperation();
+
+            op.addImage(image.getInputPathFile());
+            op.resize(image.getWidthOfFile(), image.getHeightOfFile());
+            op.rotate(image.getDegreesToRotate());
+            op.threshold(image.getWhiteBlankPercentage());
+            op.addImage(image.getOutputPathFile());
+
+            cmd.run(op);
+
+            op = new IMOperation();
+            op.size(THUMBNAILVALUE);
+            op.addImage(image.getInputPathFile());
+            op.thumbnail(THUMBNAILVALUE);
+            op.addImage(image.getOutputPathFile());
+            cmd.run(op);
+            returnTrueOrFalse = true;
+        }catch(Exception e) {
+            System.out.println(e.getMessage());
+        }finally {
+            return returnTrueOrFalse;
+        }
+    }
+
+    /**
+     * verifyDataValues are the method that validates data of file,
+     * if getWidthOfFile is equals 0 change this value for the width
+     * of file, if getHeightOfFile is equals 0 change this value for
+     * the height of file, getWhiteBlankPercentage must be a number
+     * between 0 and 100 if not setWhiteBlankPercentage takes the
+     * value of 0, DegreesToRotate must be a number between 0 and
+     * 360, if DegreesToRotate takes the value of 0.
+     * @param image variable of type Image.
+     */
+    public void verifyDataValues(ImageParam image) {
+        //filename variable obtain input path of file.
+        String filename = image.getInputPathFile();
+        //Info is class to obtain information about of File
+        Info imageInfo = null;
+        try {
+            imageInfo = new Info(filename,true);
+            if (image.getWidthOfFile() == 0) {
+                image.setWidthOfFile(imageInfo.getImageWidth());
+            }
+            if (image.getHeightOfFile() == 0) {
+                image.setHeightOfFile(imageInfo.getImageHeight());
+            }
+            if (image.getWhiteBlankPercentage() < 0 || image.getWhiteBlankPercentage() > 100) {
+                image.setWhiteBlankPercentage(0);
+            }
+            if (image.getDegreesToRotate() < 0 || image.getDegreesToRotate() > 360) {
+                image.setDegreesToRotate(0);
+            }
+        } catch (InfoException e) {
```

Comment:
check for types of throw Exceptions, because the exception can throwed for many reasons